tests/components/uvc/test_camera.py

```diff
@@ -145,12 +181,14 @@ def test_setup_nvr_error_during_indexing_nvrerror(self):
 
     def test_setup_nvr_error_during_indexing_connectionerror(self):
         """Test for error: requests.exceptions.ConnectionError."""
-        self.setup_nvr_errors_during_indexing(requests.exceptions.ConnectionError)
+        self.setup_nvr_errors_during_indexing(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'mock_uvc' in method call](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)